includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -543,7 +545,7 @@ public function process_response( $response, $order ) {
 				$order->add_order_note( $localized_message );
 				throw new WC_Stripe_Exception( print_r( $response, true ), $localized_message );
 			}
-		} else {
+		} elseif ( 'requires_payment_method' !== $response->status ) {
```

Comment:
Can you provide some context as to why this change is needed?

Reply:
If an error happens before the customer can see the boleto(like wrong cpf), stripe sends a notification that would end up putting the order on hold and reducing the stock levels. When using OXXO and Boleto the status is requires_payment_method. Other payment methods use other status so that seem the safest way to check. I'll take a deeper look at it to try to find another solution.

Reply:
I changed how the errors are handled. Now if the chosen payment method is boleto or oxxo the `update_failed_order_ajax` ajax endpoint is not called

Reply:
So the above change is no longer necessary? Please LMK when you have pushed the change.